notebook.ipynb

```diff
@@ -92,37 +92,1248 @@
+       "      <td>2021-12-04 16:00:00</td>\n",
+       "      <td>5623</td>\n",
+       "      <td>178.652435</td>\n",
+       "      <td>4.719932</td>\n",
+       "      <td>840.600281</td>\n",
+       "      <td>0.800192</td>\n",
+       "      <td>1.270305</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>104504</th>\n",
+       "      <td>2021-12-06 08:00:00</td>\n",
+       "      <td>5625</td>\n",
+       "      <td>308.520111</td>\n",
+       "      <td>1.853987</td>\n",
+       "      <td>64.687752</td>\n",
+       "      <td>1.000000</td>\n",
+       "      <td>3.222932</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>225319</th>\n",
+       "      <td>2021-09-21 07:00:00</td>\n",
+       "      <td>5651</td>\n",
+       "      <td>12.908099</td>\n",
+       "      <td>1.018451</td>\n",
+       "      <td>86.801544</td>\n",
+       "      <td>0.984353</td>\n",
+       "      <td>86.969696</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>112150</th>\n",
+       "      <td>2021-10-20 22:00:00</td>\n",
+       "      <td>5626</td>\n",
+       "      <td>257.430847</td>\n",
+       "      <td>4.842998</td>\n",
+       "      <td>542.545471</td>\n",
+       "      <td>0.902503</td>\n",
+       "      <td>0.000000</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                       date   PC4          wd        ws          blh  \\\n",
+       "232511  2021-07-17 23:00:00  5652   98.453430  2.351162   198.025925   \n",
+       "227668  2021-12-28 04:00:00  5651  273.183563  4.758546   515.898499   \n",
+       "94842   2021-10-29 18:00:00  5624  250.668884  5.661692   899.543762   \n",
+       "195467  2021-04-25 11:00:00  5644   61.174820  3.631232  1183.005981   \n",
+       "250589  2021-08-10 05:00:00  5654  233.508011  3.470477   317.930725   \n",
+       "165916  2021-12-10 04:00:00  5633  302.348145  5.649457   900.176392   \n",
+       "86944   2021-12-04 16:00:00  5623  178.652435  4.719932   840.600281   \n",
+       "104504  2021-12-06 08:00:00  5625  308.520111  1.853987    64.687752   \n",
+       "225319  2021-09-21 07:00:00  5651   12.908099  1.018451    86.801544   \n",
+       "112150  2021-10-20 22:00:00  5626  257.430847  4.842998   542.545471   \n",
+       "\n",
+       "             tcc        ssrd  \n",
+       "232511  0.000000    0.000000  \n",
+       "227668  0.584514    0.000000  \n",
+       "94842   0.996007    0.000000  \n",
+       "195467  0.925585  498.387500  \n",
+       "250589  0.513039   11.182894  \n",
+       "165916  0.997464    0.000000  \n",
+       "86944   0.800192    1.270305  \n",
+       "104504  1.000000    3.222932  \n",
+       "225319  0.984353   86.969696  \n",
+       "112150  0.902503    0.000000  "
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
```

Comment:
Add a sub title on top of each new code cell
e.g. <h4><font color=mediumvioletred>Read CSV files </font></h4>